Fix LVTs

magneticflux- opened the pull request:
Closes #165, closes #166

magneticflux- committed:
Fix ServerGamePacketListenerImpl

Closes #165, closes #166

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/architectury/architectury-api?pullRequest=167) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/architectury/architectury-api?pullRequest=167) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/architectury/architectury-api?pullRequest=167) it.</sub>

magneticflux- committed:
Fix MixinGameRenderer by giving it an explicit ordering

The Fabric API mixin uses the default priority of 1000, so we use 1100 priority to apply before it. This results in this bytecode structure, where "F" is the Fabric API injections, "A" is the Architectury API injections, and "M" is the Minecraft function being injected pre/post: `FAMAF`.

magneticflux- committed:
Fix MixinPhantomSpawner captures

magneticflux- committed:
Fix MixinPatrolSpawner captures

magneticflux- committed:
Fix MixinChunkSerializer captures

triphora commented on fabric/src/main/java/dev/architectury/mixin/fabric/client/MixinGameRenderer.java:
```diff
@@ -39,7 +39,7 @@
 import java.util.function.Consumer;
 
-@Mixin(GameRenderer.class)
+@Mixin(value = GameRenderer.class, priority = 1100)
```
What was the purpose of changing the priority here?

magneticflux- replied:
I talk about why in https://github.com/architectury/architectury-api/pull/167/commits/3d2a81f80745eca08ec1957005062a1d1f19f429

magneticflux- replied:
If it's <= 1000, it crashes because it performs a modification (the cancellation I think?) that is incompatible with the Fabric API modification later on.
